crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -39,6 +39,7 @@ pub struct Diagnostic {
     sum: f64,
     max_history_length: usize,
     pub is_enabled: bool,
+    pub num_of_decimals: usize, // Seams like it needs to be usize
```

Comment:
```suggestion
    pub num_of_decimals: usize, // Seems like it needs to be usize
```

Reply:
Well that was a little bit embarrassing but it has been fixed now! ;)